src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -682,6 +683,7 @@ bool Dataloader::_load_map_dir(GameManager& game_manager) const {
 	static constexpr std::string_view default_region_sea = "region_sea.txt"; /* Some empty province sets (unused). */
 	static constexpr std::string_view default_province_flag_sprite = "province_flag_sprites"; /* Canal sprite/model names. */
 	static constexpr std::string_view climate_file = "climate.txt";
+	static constexpr std::string_view region_colours = "colors.txt"; /* File from interface_directory */
```

Comment:
```suggestion

	static constexpr std::string_view region_colours = "interface/colors.txt"; /* This file is not in map_directory */
```
As explained above, might as well just put the whole path in this variable. Also adding a newline between this and the filenames above would help make clear that it's not in the `map` directory.